front-end/src/style.css

```diff
@@ -1,19 +1,32 @@
-@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Concert+One&family=DynaPuff&family=Inter&display=swap');
 
 :root {
-  /*Screen background color, button text color*/
-  --bg: #ffffff;
-  /*All text color except button text*/
-  --text: #000000;
-  /*Button background colors*/
-  --accent: #757de8;
-  color: var(--text);
-  background-color: var(--bg);
+  --bg: #FFFFFF;
+  --main-rgb: 255, 128, 0;
```

Comment:
all these CSS variables allow for easy swapping - change in one place, and the whole game should change to match